### DISCUSSION TITLE
LaserGRBL v4.9.4 Etching DOUBLE IMAGE

### DISCUSSION THREAD

**Discussion by aprilajune** [arkypita/LaserGRBL#2072] [General]
The laser was working fine. Etching just as it should, then it started acting like it was sticking, then releasing, sticking, releasing. We changed out the motor cables and now, it is running smoothly except for the double image. It's not a cleancut image. On stars, it looks like double stars one over the other. We tried calibrating it according to the PDF but that's not working either. We need some help here. I see most of the questions are unanswered so I am hoping someone will tell us how to fix this problem. 
![IMG_20230520_155731849](https://github.com/arkypita/LaserGRBL/assets/134170176/fbd7132a-6d20-427b-8e39-c30a58aad712)

**Comment by Tribo-2dot0** [comment]
Isso é problema mecânico. Verifique o aperto correto das correias e polias dentadas. Verifique também a integridade dos dentes das mesmas.
